ui-frontend/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/utility.js

```diff
@@ -14,16 +14,24 @@
  **/
 
 const _ = require('underscore')
-const Openlayers = require('openlayers')
+
+import { transform } from 'ol/proj'
+import { boundingExtent } from 'ol/extent'
+import { getCenter } from 'ol/extent'
+
 const properties = require('../../../../js/properties.js')
 
 function convertPointCoordinate(point) {
   const coords = [point[0], point[1]]
-  return Openlayers.proj.transform(coords, 'EPSG:4326', properties.projection)
+
+  const transformation = transform(coords, 'EPSG:4326', properties.projection)
+
+  return transformation
 }
 
 function unconvertPointCoordinate(point) {
-  return Openlayers.proj.transform(point, properties.projection, 'EPSG:4326')
+  const transformation = transform(point, 'EPSG:4326', properties.projection)
```

Comment:
Should the source and destination be fliped as in original e.g. `transform(point, properties.projection, 'EPSG:4326')`

Reply:
Yep, you are correct. Good catch.